src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -128,6 +128,14 @@ public class TestConnectionCommand : PSCmdlet
         public Int32 BufferSize { get; set; } = 32;
 
         /// <summary>
+        /// The following is the definition of the input parameter "BufferSize".
+        /// Buffer size sent with the this command. The default value is 32.
+        /// </summary>
+        [Parameter]
+        [ValidateRange((int)1, Int32.MaxValue)]
+        public Int32 TimeOut { get; set; } = 1000;
```

Comment:
>1000 [](start = 45, length = 4)

I'm a little worried about this - how did you land on this value? Could the default value be Int32.MaxValue to maintain current behavior? Anyone using this new parameter would likely provide a much smaller number in any case